origin-faucet/migrations/20190214183422-faucet_txn.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const tableName = 'faucet_txn'
+
+const enums = require('../src/enums')
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable(tableName, {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      campaign_id: {
+        type: Sequelize.INTEGER
+      },
+      status: {
+        type: Sequelize.ENUM(enums.FaucetTxnStatuses)
+      },
+      from_address: {
+        type: Sequelize.STRING
+      },
+      to_address: {
+        type: Sequelize.STRING
+      },
+      amount: {
+        type: Sequelize.DECIMAL
+      },
+      currency: {
+        type: Sequelize.STRING
+      },
+      txn_hash: {
+        type: Sequelize.STRING
+      },
+      created_at: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updated_at: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    }).then(() => queryInterface.addIndex(tableName, ['campaign_id']))
```

Comment:
we can also add `to_address` to index, since it will also be queried by that filter

Reply:
Yes, good catch.